python/revng/ptml/__init__.py

```diff
@@ -0,0 +1,85 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import xml.sax
+from contextlib import contextmanager
+from dataclasses import dataclass
+from typing import Callable, ContextManager, List, Optional, Union
+
+from .printing import PrinterBackend
+from .sax import Printer, Splitter
```

Comment:
I wouldn't re-export these.

Reply:
There's the `__all__` attribute which says which symbols are re-exported